WTWSMS/common/traits/02_traits.txt

```diff
@@ -1093,10 +1095,15 @@ schemer = {
 theologian = {
 	lifestyle = yes
 	
-	learning = 2
-	intrigue = 1
+	learning = 4
+	diplomacy = 2
 	
 	ruler_designer_cost = 10
+	church_opinion = 15
+
+	opposites = {
+		cynical
+	}
```

Comment:
It's an improvement from the Vanilla rendition. Surely, you don't agree with everything Paradox laid down, right?